config/containers/logging/configure.md

```diff
@@ -157,9 +151,25 @@ see more options.
 | [`etwlogs`](etwlogs.md)       | Writes log messages as Event Tracing for Windows (ETW) events. Only available on Windows platforms.           |
 | [`gcplogs`](gcplogs.md)       | Writes log messages to Google Cloud Platform (GCP) Logging.                                                   |
 | [`logentries`](logentries.md) | Writes log messages to Rapid7 Logentries.                                                                     |
+| [`local`](local.md)           | Logs are stored in a custom format designed for minimal overhead.                                             |
 
 ## Limitations of logging drivers
 
-The `docker logs` command is not available for drivers other than `json-file`
-and `journald`.
+- The `docker logs` command, which provides support for the local logging driver, was added in Docker CE version 17.05. The `docker logs` command is only available on the following drivers:
+    - `json-file`
+    - `journald`
+    - `local`
+
+Refer to [Reading logs when using remote logging drivers](/dual-logging) for information about using `docker logs` to read container logs locally for many third party logging solutions, including: 
+
+    - syslog	
+    - gelf	
+    - fluentd	
+    - awslogs	
+    - splunk	
+    - etwlogs	
+    - gcplogs	
+    - Logentries
 
+- Reading log information requires decompressing rotated log files, which causes a temporary increase in disk usage (until the log entries from the rotated files are read) and an increased CPU usage while decompressing. 
+- The capacity of the host storage where docker’s data directory resides determines the maximum size of the log file information.
```

Comment:
@anusha-ragunathan - Can you PTAL?

Reply:
I agree with @thaJeztah. `local` logger has default max file size and max file count. So its not gonna consume all your disk space.

Regarding recommending `local` logging driver, I remember a conversation back during implementation about why having "local" driver as default would break compatibility. @thaJeztah : do you remember why? Wondering if that reason could prevent us from recommending "local" logger.

Reply:
@anusha-ragunathan Thank you! @thaJeztah - PTAL - once this is resolved, we can ship this! Thank you both so much again for your input here!